README.md

```diff
@@ -1,5 +1,18 @@
 # thesis-travel-app
+
 Travel App
 
 Team: OperationDeploy
 Members: Austin, Garrett, Jessa, Josh
+
+HOW TO DEPLOY APP:
```

Comment:
You are starting the front end and back end servers, not deploying. Please change this wording.